modules/generators/messages/randsom_note/secgen_metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+
+<generator xmlns="http://www.github/cliffe/SecGen/generator"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.github/cliffe/SecGen/generator">
+
+  <name>Ransom Note Generator</name>
+  <author>Jack Biggs</author>
+  <module_license>MIT</module_license>
+  <description>Creates a file in which a ransom note resides.</description>
+
+  <type>string_generator</type>
+  <type>local_calculation</type>
+  <platform>linux</platform>
+  
+  <read_fact>name</read_fact>
+
```

Comment:
Maybe provide a default input for the name, or update the template so it still looks good without a name. Not a blocker.